machines/vcItem.ts

```diff
@@ -1331,14 +1349,15 @@ export const createVcItemMachine = (
   serviceRefs: AppServices,
   vcKey: string
 ) => {
-  const [, idType, id, requestId, isPinned] = vcKey.split(':');
+  const [, idType, hashedId, requestId, isPinned, id] = vcKey.split(':');
```

Comment:
is `id` at the end in UIN or VID?

Reply:
yes, as we have changed the  " VC_ITEM_STORE_KEY =`vc:${vc.idType}:${vc.hashedId}:${vc.requestId}:${vc.isPinned}:${vc.id}`;  "
the id is at the end